tests/rptest/tests/simple_e2e_test.py

```diff
@@ -69,3 +81,60 @@ def test_consumer_interruption(self):
         assert error is not None
         assert "Consumed from an unexpected" in str(
             error) or "is behind the current committed offset" in str(error)
+
+    @skip_debug_mode
+    @cluster(num_nodes=6)
+    def test_leader_acks(self):
+        ev = threading.Event()
+
+        def inject_failures():
+            fi = FailureInjector(self.redpanda)
+            while not ev.is_set():
+                range = self.redpanda.nodes[0:1]
+                if random.randint(0, 100) > 50:
+                    range = self.redpanda.nodes[1:2]
```

Comment:
nit: there is just 1 node in each range, right? Can't we just choose randomly from nodes[0:2] instead?

Reply:
actually my bad, i wanted to do the two nodes